### PR TITLE
Allow 6.x versions of FHIR

### PR DESCRIPTION
**Pull request by mint-thompson**
**Description:**
FHIR version 6 is currently in balloting stages. It does not appear that any code changes are needed yet for allowing an R6 IG. The usual warning about prerelease versions will be shown when one of the ballot versions is used.

**Testing Instructions:**
The main question here is whether supporting R6 is actually as simple as I think it is. Having looked over [information about the R6 ballot changes](https://hl7.org/fhir/6.0.0-ballot2/ballot-intro.html), I don't think there's anything that will need special handling. 

The new datatype [RelativeTime](https://hl7.org/fhir/6.0.0-ballot2/datatypes.html#RelativeTime) is complex enough that it doesn't make sense to try to support assigning it in a single rule. But, it may be worth considering if we want to support checking [its invariants](https://hl7.org/fhir/6.0.0-ballot2/datatypes-definitions.html#RelativeTime). rlt-1 is a very simple check, and rlt-2 isn't too much more complicated. We do something like this for Extensions for their [ext-1 invariant](https://hl7.org/fhir/extensibility-definitions.html#Extension).

**Related Issue:**
#1485

### CODE REVIEW

**Review by cmoesel — DISMISSED** [review]
Well, that was easier than expected! I tested it out by creating an R6 IG and profiling an element that doesn't exist in R5. It worked! I was even able to run the IG Publisher after.

One of the reasons this is so simple now is that past us thought ahead to this moment and added this: https://github.com/FHIR/sushi/blob/04b9d7beb12ad99489e03d0feeb142cf79c09762/src/utils/FHIRVersionUtils.ts#L61-L75

Anyway, nice job!

File: test/utils/Processing.test.ts [hunk 1]
```diff
@@ -426,6 +426,12 @@ describe('Processing', () => {
       expect(config.fhirVersion).toEqual(['4.5.0']);
     });
 
+    it('should allow FHIR R6', () => {
+      const input = path.join(__dirname, 'fixtures', 'fhir-r6');
+      const config = readConfig(input);
+      expect(config.fhirVersion).toEqual(['6.0.0-ballot2']);
+    });
```
Comment (jafeltra) [inline test/utils/Processing.test.ts:433]: I don't know what the difference is, but I was also able to use just `6.0.0` as a `fhirVersion` already. Is it worth adding a test for that? SUSHI recognizes it as a non-preRelease version, thanks to the snippet of code Chris linked.

**Review by jafeltra — COMMENTED** [review]
I just left one question about a test, but I might be getting ahead of myself, so let me know what you think.

File: test/utils/Processing.test.ts (hunk 1 above)
Comment (mint-thompson) [reply test/utils/Processing.test.ts:433]: Getting a package downloaded for `6.0.0` right now is not so good, but either way, I can add a test for when that version is specified.

**Review by jafeltra — APPROVED** [review]
Looks good to me!

**Review by cmoesel — APPROVED** [review]
Re-approving!

**Pull request merged**